pkg/microservice/aslan/core/common/repository/models/workflow.go

```diff
@@ -309,6 +309,7 @@ type HookPayload struct {
 type TargetArgs struct {
 	Name             string            `bson:"name"                      json:"name"`
 	ServiceName      string            `bson:"service_name"              json:"service_name"`
+	ServiceType      string            `bson:"service_type,omitempty"    json:"service_type,omitempty"`
```

Comment:
omitempty？ Is this a required parameter?

Reply:
optional